crawl-ref/source/item-use.cc

```diff
@@ -1845,9 +1847,9 @@ bool puton_ring(int slot, bool allow_prompt)
     if (prompt_failed(item_slot))
         return false;
 
-    bool prompt = allow_prompt ? Options.jewellery_prompt : false;
+    bool prompt = allow_prompt ? Options.jewellery_prompt : false; 
```

Comment:
Extra space at the end of the line here; you can use `git diff --check` to check for problems like this.